### ISSUE TITLE
Luna 1.0 "Aero" Shiplist

### ISSUE THREAD

**Issue by Studio384** [GetLuna/Luna#863]
It has been great, our first generation forum software (ModernBB 1.6, 1.7) and second generation (ModernBB 2.x, 3.x) have grown to contain new features, improved capabilities and so much more. But now, it's time for the third generation to let it do its job, and in the form of Luna 1.0. This is our shiplist to the next major revision of our forum software. And it is going to be huge! A new Backstage, a new Mainstage, a new installation, a new code base. And of course, a bunch of new features!

# Tickets

## Mainstage
The Mainstage (our brand new name to replace "front-end") will be build op from scratch - of course - and won't be like anything you've ever seen. With Luna, we want to reimagen board software, and for now, we're doing a prety good job.

### Board
- [x] #901 **Redirect sections**: redirect sections are no longer available.
- [x] #953 **First Run enhancements**: the First Run experience has been improved with better design and new abilities.
- [x] #1000 **Subforum support**: Luna includes support for subforums.

### Commenting
- [x] #869 **New editor**: the new editor in Luna will be much like the editor in ModernBB, through we want to give it a slimmer design and some new functionality.
- [x] #885 **BBCode handling**: the new editor in Luna should act smarter then the one in ModernBB.
- [x] #893 **Use emojis**: Luna will use the UTF-8 emojis in the used OS instead of image based icons.
 - [x] #1069 **Default emoji size**: the default size for emojis has been changed from 14px to 16px.
 - [x] #1220 **Space**: before and after an emoticon, Luna will now add some space.
- [x] #963 **Code Highlighter**: we will use PrismJS to give your code a fancy color.
- [x] #1065 **Smilies**: you can now choose to use emoticons or emojis.
- [x] #1102 **Emoticon redesign**: Luna contains newly designed emoticons.
- [x] #1210 **Accessability:** Sticky topic and Muted edit now have a tooltip.

### Users
- [x] #870 **Profile revamp**: the Profile has been revamped completely to be more to-the-point, and contain all new features with awesomeness.
- [x] #904 **User search**: the new user search feature on the user list is now simpeler, easier and smaller.
- [x] #961 **Luna Inbox**: the private messaging system Luna Inbox has been added.
- [x] #985 **Change email**: the "Change email" interface has been drasticaly improved.
- [x] #989 **Clear description**: we made the "Post" and "Topic" setting in the profile more clear.
- [x] #1028 **Profile settings revamp**: the profile settings have been redesigned completely and all settings are now brought together in one single page, all ready to be saved at once.
- [x] #1080 **Improve Inbox**: make it part of the profile, remove some features we don't need.

### Login + registration

- [x] #910 **Rules**: rules will no longer have their own page, instead, they will be available on the help page.
- [x] #930 **Login everywhere**: Luna now shows the login form on top of the open page instead of loading a new page.
- [x] #937 **Language at registration**: the user's language can no longer be set on registration.
- [x] #977 **Stronger passwords**: Luna now generates longer passwords.
- [x] #1039 **Backstage login**: the Backstage will get it's own login form in case something happens to the Mainstage login form.

### Management
- [x] #1009 **Soft delete**: you can now delete a post from user's view without actually deleting it.

### Notifications
- [x] #954 **Notifications**: add support for notifications. They should be triggered at some events.
- [x] #1036 **Fly-out option**: admins can now disable the fly-out to boost performance.
- [x] #1037 **Notification events**: certain events now trigger notifications.
- [x] #1040 **Profile split-off**: the notifications page is no longer part of the profile file.
- [x] #1140 **Notification behavior**: the behavior of the page has been altered to show your notifications when no id is set in the url.

### Other features
- [x] #951 **Profile color**: users can now set a color for their profile if the theme supports it.
- [x] #965 **Backstage in user menu**: the Backstage item has been removed from the main menu in favor of it being relocated into the user menu.
- [x] #1011 **Back to top**: the footer now has a "Top" link to put you back to the top of the page.
- [x] #1012 **Search from everywhere**: you can now search from every page.
- [x] #1032 **Improved heading**: the heading has been improved with some additions.

## Backstage
The Backstage will feature an all-new design, with more focus on management of your board. It will, of course, also contain all new features introduced in Luna 1.0.

### General
- [x] #884 **Create new Backstage interface**: the new Backstage interface should be clean and simple, might look a little bit like the current ModernBB version...
 - [x] #903 **Backstage Accents**: the Backstage Accent colors will be gone for this release. Maybe that we will add them again later on.
 - [x] #919 **Interface style guidelines**: we now are following some interface style guidelines in Backstage.
 - [x] #974 **Touch it**: the navigation bars now handle touch better on non-small touchscreens (phablets, tablets, desktops).
 - [x] #1006 **Small screens**: the support for small screens in the Backstage has been improved.

### Backstage
- [x] #898 **New Backstage index**: the Backstage index has been improved with a better formation.
 - [x] #895 **System info**: the "Server statistics" have been replaced with the more extended "System info".
 - [x] #899 **Admin notes**: the Backstage now has an "Admin notes" section which can be edited by all admins.
 - [x] #1164 **Default note**: Luna will no longer add a note to Admin notes by itself.
- [x] #999 **First Run**: the Backstage now shows a "First Run" panel upon opening the Backstage area for the first time until the message is dismissed.
- [x] #1005 **Release notes**: the release notes of Luna are now included in the Backstage.
- [x] #1008 **Writeable config warning**: if the config file is writeable, the Backstage will warn for that issue.
- [x] #1092 **Check for updates**: the Check for update system has been removed, it should make its return now.
- [x] #1104 **Branch options**: it's now possible to check for which branch you want to receive updates.

### Content
- [x] #962 **Moderation tools**: the moderation tools, that used to be part of the Mainstage, are now placed into the Backstage.

### Users
- [x] #1021 **User tools**: a new page, "Tools" has been added and includes feature you can use on your users.

### Settings
- [x] #865 **Revamp menu system**: a revamp to the menu system should make the thing more user-friendly in general.
- [x] #915 **BBCode disabling**: it is no longer possible to disable the usage of BBCode anywhere on the forum, disabling img-tag usage is still possible, through.
- [x] #918 **Ranks management**: ranks can be saved all at once, instead of one by one.
- [x] #967 **Inbox options**: the Inbox settings should be placed under Features and Permission instead of having it's own page. We don't want a bunch of pages with 3 options.
- [x] #1022 **Copyright options**: you can alter the copyright notice on your forum with some small options.
- [x] #1024 **Emoji settings**: you can change the size of emojis from the Backstage.
- [x] #1025 **Back to top**: enable or disable "Back to top" on the board.
- [x] #1035 **Disable search**: you can now disable the search bar in the navbar.
- [x] #1038 **Footer statistics**: you can now disable the footer statistics.
- [x] #1073 **Theme improvements**: the Theme settings page has been improved with altered features.

### Maintenance
- [x] #1090 **Maintenance split off**: Maintenance is now a separate menu option in the Backstage.
- [x] #1091 **Notifications Prune**: you can now clean up Notifications from the Backstage interface.

### Installation and upgrade
- [x] #887 **Object oriented coding**: the installer is now more focused on object oriented coding to improve readability of the code, and of source, make it smaller.
- [x] #914 **Update warning**: when updating Luna, it will warn you to not update a ModernBB installation in a productive environment.

### Language
- [x] #1068 **Translation improvements**: make it possible to translate all strings again.
- [x] #1093 **Language strings**: some language strings didn't appear where they should.

### Themes
With the new themes engine, Random and its variantions will have to go (we might rebuild them in the future, just for fun). Here you can find all tickets about the default theme "Sunrise", and the new theme engine.

#### Engine

- [x] #902 **Section color**: the section settings now display a field to add a color to the section, optionaly used by themes.
- [x] #905 **Logic and markup split**: step 1 in the new engine: moving logic and markup away from each other. This is one of the more important tickets in the whole Luna 1.0 milestone.
- [x] #908 **Theme settings revamp**: the settings page in the Backstage is revamped with a bunch of new features and more.
- [x] #909 **Settings cleanup**: settings that now belong to themes themselves have been removed from the Core.
- [x] #1076 **Theme parents**: themes can now have a parent of which they take all pages and styles unless defined otherwise.
- [x] #1078 **General improvements**: this is the ticket where we store all Theme engine related commits.
- [x] #1094 **Rename themes folder**: to prevent issues, we'll give the style folder the name "themes".
- [x] #1219 **Responsive screenshots**: theme screenshots are now responsive.

#### Luna (theme)

- [x] #900 **Default theme**: our new default theme, Luna, will replace Random, target is to be modern, light and simple.
 - [x] #862 **Revamp index**: the index can be made much more usefull, instead of listing all sections, it will also display the recent active threads. This will render a "section view" useless.
 - [x] #912 **Thread interface**: the new thread interface brings more focus to the content.
 - [x] #950 **Profile interface**: the new profile interface brings an improved design to the user.
 - [x] #1097 **Pagination divider**: there is now a clear pagination divider.
 - [x] #1101 **Minor theme improvements**: we've improved the default design on a couple of points.
 - [x] #1217 **New post accent**: we've updated the new post indication.
- [x] #968 **Profile color usage**: the profile color is now used throughout the whole Mainstage.
- [x] #1056 **Responsive design**: with the main design finished, we should make it responsive and mobile ready too.

#### Other themes
- [x] #1075 **Sunrise theme**: Sunrise is a variation upon Luna (the theme) with a more classic structure. It's a child theme for Luna.

### Frameworks + components
Luna makes use of a couple of frameworks and components, Bootstrap, Font Awesome and jQuery to name some of them.

- [x] #896 **TrentUI Framework**: we now include the TrentUI Framework in the package for both Backstage and the default Mainstage theme.
- [x] #897 **Font Awesome 4.1**: Font Awesome version 4.1 will replace Glypicons.
- [x] #906 **Font Awesome 4.2**: Font Awesome 4.1 has been updated to Font Awesome 4.2.
- [x] #955 **Bootstrap 3.3**: Bootstrap 3.2.0 has been updated to Bootstrap 3.3.0.
- [x] #966 **Bootstrap 3.3.1**: Bootstrap 3.3.0 has been updated to Bootstrap 3.3.1.
- [x] #1020 **jQuery 2.1.3**: jQuery 2.1.1 has been updated to jQuery 2.1.3.
- [x] #1067 **Bootstrap 3.3.2**: Bootstrap 3.3.1 has been updated to Bootstrap 3.3.2.
- [x] #1074 **Font Awesome 4.3**: Font Awesome 4.2 has been updated to Font Awesome 4.3.
- [x] #1105 **Bootstrap 3.3.4**: Bootstrap 3.3.2 has been updated to Bootstrap 3.3.4.

### Other

- [x] #916 **Coding conventions**: with Luna 1.0, we're going to put some focus on coding conventions.
 - [x] #1030 **Selected**: use `selected` instead of `selected="selected"`.
- [x] #932 **Users online counter**: "Users online" will stay up-to-date better.
- [x] #936 **Embedded YouTube**: support for embedding YouTube videos has been improved.
- [x] #956 **Database enhancements**: the database now works much more efficient on some points.
- [x] #960 **Icon usage**: Luna uses more icons in both Backstage and Mainstage for visual appeal.
- [x] #1029 **Brand update**: a long run ticket to update our brand from ModernBB to Luna.
- [x] #1034 **Image optimalisation**: images are optimized to take as less space as possible.
- [x] #1081 **CDN usage**: with the end of previews coming near, it's time to move back to CDNs.
- [x] #1122 **Back-up names**: Luna will now call its back-ups "luna" instead of "modernbb".

### Features
- [x] #1062 **Remove install**: Luna will ask the admin to remove the install file after the installation has been finished.

### Fixes
- [x] #939 A security issue has been fixed.
- [x] #943 A flaw in redirection has been fixed.
- [x] #984 Fixes a security flaw.
- [x] #1050 Notifications are no longer accessable by everyone.
- [x] #1063 Security vulnerability in install.php.

## Round-up tickets
These tickets are round-ups, they contain multiple tickets that have been issued in an earlier, unmerged, release of Luna.

- [x] #992 **ModernBB 3.6 Blue**: the tickets from ModernBB 3.6 Blue, the final ticket before Preview 1.
- [x] #1031 **Luna 1.0 Aero**: the final ticket that contains all preparations that need to be done before we can release the stable Luna 1.0 Aero release.

## Bugfixes

- [x] #911 Fixes a bug that prevents db_update.php from loading.
- [x] #922 Admin group permissions won't get altered in the database anymore.
- [x] #931 jQuery will no longer fail to load on an SSL Certified server.
- [x] #933 The newest topic title on the index will be updated too when editing a title.
- [x] #934 "Last edited by" is displayed when asked for it.
- [x] #935 Allignement issues in search have been fixed.
- [x] #938 Fix issue that prevented the default avatar to be displayed when another avatar directory was selected.
- [x] #940 ModernBB will check for the right location when checking for updates again.
- [x] #945 The editor will no longer add 2 tabs for no reason.
- [x] #946 The editor will now remember the content when going in preview mode.
- [x] #948 An issue with the URL and IMG tag being used at once has been fixed.
- [x] #949 A search will no longer return an error.
- [x] #952 We removed a pointless if-statement.
- [x] #957 The menu page no longer accept all random input.
- [x] #969 It's possible to edit posts again, fixes another issue with edits too.
- [x] #970 Video width and height setting where displayed in each others field.
- [x] #971 Issue with the responsive design of videos has been fixed.
- [x] #986 The avatar file size field no longer shows "pixels".
- [x] #991 An bug in validate_redirect() has been fixed.
- [x] #998 Avatar uploads are possible again.
- [x] #1004 Remove setting of values in quickpostform.
- [x] #1013 The login modals now also change color according to the used Mainstage color.
- [x] #1014 The avatar in Me is now properly sized again.
- [x] #1015 Light themes have white :focus style for navbar elements.
- [x] #1016 Fix issue with login form and usage of keyboards.
- [x] #1017 A bug with the database prefix has been fixed.
- [x] #1018 Guest have no longer access to Inbox.
- [x] #1026 New users now have a timezone included in their account by default.
- [x] #1033 The forum list now aligns correctly.
- [x] #1041 The "Mark as read" button has been fixed.
- [x] #1042 The "be the first one"-link in the index won't show when not required.
- [x] #1043 We've fixed the title of post.php when posting a new post.
- [x] #1044 A couple of markup issues have been fixed.
- [x] #1045 Post.php shows the correct title when posting a post again.
- [x] #1046 First run can be disabled again.
- [x] #1047 "Users online" is readable again when the menu is opened.
- [x] #1048 The profile signature panel now shows the signature again.
- [x] #1049 The "Admin"-pane bug has been fixed.
- [x] #1051 Moving a post no longer triggers an error.
- [x] #1052 Users can quote comments again.
- [x] #1053 Quotes no longer apply the footer style to themselves.
- [x] #1054 The "Subscribe" link has been added again to the sidebar.
- [x] #1055 Users can edit their comments again.
- [x] #1057 Some links and spelling errors have been fixed.
- [x] #1058 The First Run password link now opens the modal again.
- [x] #1059 Login.php is included after the DOCTYPE.
- [x] #1060 Thread.php no longer loads without permission.
- [x] #1066 A duplicated search id has been removed in help.php.
- [x] #1070 A wrong link in the Backstage has been fixed.
- [x] #1072 A bug in the userlist with long names has been fixed.
- [x] #1077 The search button isn't sized correctly.
- [x] #1079 We now link to me.php everywhere, instead of profile.php.
- [x] #1095 An issue with new posts is fixed.
- [x] #1096 Announcements now work correclty again.
- [x] #1106 A bug with misaligned icons in the navbar has been fixed.
- [x] #1107 A bug with unreadable messages in the footer has been fixed.
- [x] #1108 The markup on the users page has been fixed.
- [x] #1109 The code above the HTML-tag in Settings have been relocated.
- [x] #1110 Search no longer shows all posts to be from closed topics.
- [x] #1112 Fix multiple issues with translations.
- [x] #1113 Check for correct permissions when showing delete and soft delete.
- [x] #1114 Fixes a bug when registrating with rules enabled.
- [x] #1115 Fix undefined index in Inbox.
- [x] #1116 Fix undefined index in Notifications.
- [x] #1117 Fix undefined redirect_url.
- [x] #1118 Fix advanced search issues.
- [x] #1119 Fix issue with Database tools.
- [x] #1121 Remove outdated link to maintenance mode.
- [x] #1123 Fix issues with IP stats.
- [x] #1124 Fix bug in Ban page that loads nav twice and defines current page twice.
- [x] #1125 Fix issue that caused deleted users not to be deleted.
- [x] #1126 Fix undefined variable in search.
- [x] #1127 Fix undefined redirect_url variable.
- [x] #1128 Fix unindentified index level.
- [x] #1129 Fix undefined variable lang in search.
- [x] #1130 Fix missing variables when posting a new post.
- [x] #1131 Fix issues with the moderation tools.
- [x] #1132 Fix issues with the moderation tools when closing topics.
- [x] #1133 Fix undefined variable when creating a new post.
- [x] #1134 Fix undefined message variable when editing a post.
- [x] #1135 Fix undefined index in Inbox.
- [x] #1136 Fix issue when deleting post from Inbox.
- [x] #1137 Fix wrong link for dates in Inbox.
- [x] #1138 Fix abbility to delete items from Inbox.
- [x] #1139 Fix issue with login redirect url.
- [x] #1141 Quoting a message in Inbox didn't quote that message.
- [x] #1142 An issue on the ban page has been fixed.
- [x] #1143 Fix undefined index Database.
- [x] #1144 Fix undefined variable item_status.
- [x] #1145 Fix undefined index visible.
- [x] #1146 Put space between attributes.
- [x] #1147 Fix errors when trying to restore a back-up without choosing a file.
- [x] #1148 The moderator links in forum views have been fixed.
- [x] #1149 Fix issues when pruning the board.
- [x] #1150 Fix issues that appear when starting a back-up.
- [x] #1151 Added missing "=" in link.
- [x] #1152 Remove variable that isn't used from forum.
- [x] #1153 Fix undefined index when posting new item.
- [x] #1154 Fix undefined variable in Inbox.
- [x] #1155 Fix another undefined varibale in Inbox.
- [x] #1156 Fix issue when moderating a topic.
- [x] #1157 Fix when opening, closing, etc. a topic from post.
- [x] #1158 Fix bug in parser with emoticons/emojis.
- [x] #1159 Don't provide option to edit posts in PM.
- [x] #1160 Custom copyright text is now escaped.
- [x] #1161 Fix undefined index last_session_request.
- [x] #1162 Fix undefined idnex redirect_url.
- [x] #1165 Request password now has error handling.
- [x] #1166 Pruning post no longer displays results above html-tag.
- [x] #1167 Fix undefined variable url.
- [x] #1168 Fix 404 not found error when splitting topics.
- [x] #1169 Fix loading twice when deleting posts.
- [x] #1170 Fix undefined variable redirecct_url.
- [x] #1171 Fix loading twice when searching IP address without IP address.
- [x] #1172 Fix undefined variable forum_name.
- [x] #1173 Fix set moderator access.
- [x] #1174 Fix issues when guests use the post form.
- [x] #1175 Fix forgotten password link form_sent.
- [x] #1176 Fix incorrect focus and tab order.
- [x] #1177 Fix missing ```<b>```-tag for topic view.
- [x] #1178 Fix blank page when sending an email.
- [x] #1179 Fix admin note escape.
- [x] #1180 Fix loading Backstage menu twice in settings.
- [x] #1181 Fix missing color field for forums when updating.
- [x] #1182 Fix undefined index errors in Inbox.
- [x] #1183 Fix undefined index.
- [x] #1184 Fix inability to fetch forum information.
- [x] #1185 Do not grand members the permission to manage soft deleted posts.
- [x] #1186 Fix language error.
- [x] #1187 Fix issue when accessing the login form directly.
- [x] #1188 Fix undefined variable cur_index.
- [x] #1189 Fix undefined index o_admin_note.
- [x] #1190 More consistent font stacks for Backstage and Mainstage.
- [x] #1191 Fix loading twice when adding an empty rank.
- [x] #1192 Fix undefined index redirect_url in First Run.
- [x] #1193 Fix inability to add a new group.
- [x] #1194 Fix escape menu item name.
- [x] #1195 Fix undefined varibales smilies.
- [x] #1196 Fix bug that causes threads from locked forums to show up anyway.
- [x] #1197 Do not allow video tags in signatures.
- [x] #1198 Do not show Inbox in profile menu when Inbox is disabled.
- [x] #1199 Fix untranslatable string.
- [x] #1200 No access to email form.
- [x] #1201 Fix preview mode removing message from editor.
- [x] #1203 It's no longer possible to create empty entries in Censoring.
- [x] #1204 It's no longer possible to create empty entries in Categories.
- [x] #1205 Fix a missing "to".
- [x] #1206 Fix escape menu name and url.
- [x] #1207 Do not allow empty menu items.
- [x] #1208 Fix missing closing tag.
- [x] #1209 Do not show the post button to people without permission.
- [x] #1211 Fix issue when restoring the board.
- [x] #1212 Fix wrong description for rules option.
- [x] #1213 Fix smilies option still available in Backstage.
- [x] #1214 Fix maintenance mode undefined variable.
- [x] #1215 Fix notification fly-out style.
- [x] #1216 No longer show arrow when show online users is disabeld.
- [x] #1221 Fix undefined variable cur_index.
- [x] #1222 Fix top alignment in First Run.
- [x] #1223 Fix for loosing guest information when in Preview mode.
- [x] #1224 Multiple language strings are now translatable.
- [x] #1225 Do not load the admin nav.
- [x] #1226 Do not let editor lose focus when toolbar is used.
- [x] #1227 We've fixed a couple of spelling mistakes.
- [x] #1228 Fix cut of replies count with long topic titles.
- [x] #1230 Fix loading admin nav.
- [x] #1233 Fix an issue with the file picker.
- [x] #1236 Fix invalide address for moved posts.
- [x] #1242 Fix error with random digits in the userlist.

## Others
Other stuff we have to get done but isn't related to the Luna Core.

- [x] #867 **Write documentation**: with new software comes a new documentation.

# Commits
Notable changes that do not have a ticket are listened below.

### General
- [x] 2eb0ac1 The Backstage now has less whitespace at the end of panels.
- [x] b23f563 The logic behind the rank update has been improved.
- [x] 46e73cb The Backstage User menu has been extended to match the Mainstage's version.
- [x] c9c4439 We've made small changes to the menus in the Backstage.

### Settings
- [x] 1c7e311 Code from Menu page in Backstage has been improved.
- [x] 804b9b6 The obsolete post_responsive setting has been removed.

### General enhancements
- [x] 6b0ff5c The update check now only checks for o_core_version, as its always different.
- [x] ad7eb37 There are new colors added for forums to choose from.
- [x] 631b804 "moved_to" entries are no longer shown on the index page when no id is set.

### Installation and upgrade
- [x] e2df0ce New db_update.php design, matching login.php for Backstage.

### Language
- [x] 2f9e18d The pms_plugin.php language files has been removed.
- [x] e6de6d3 The language files have been merged into one file.

### Themes
- [x] b87cfc5 The "Choose user group" interface has been improved.
- [x] 616e7ed The cancel button now goes back to index instead of viewforum.
- [x] c778631 Improved design for edit.php, based on post.php.

### Others
- [x] 311c84a The pagination function has been updated to require less code.
- [x] 17bc04a The test image has been updated to Luna.
- [x] 7e0cfb3 zSettings has been removed from the Backstage.

## Bugfixes
- [x] f5e4369 Fix bug for section setting unable to be loaded due to double query.
- [x] 169fc6b Fix for footer being loaded twice in the section settings page.
- [x] 2c4e8e0 Fix issue with Maintenance settigns failing to load.
- [x] ab75a47 Fix a bug when accessing sub pages in Me.
- [x] 439c003 The Topic view now works correctly again.
- [x] a863ae4 The category and forum list now is shown again in the "Move topic" dialoge.
- [x] 59372f3 Also fix the merge moderation tool.
- [x] 1bd24d5 Missing errors in the updates have been added.
- [x] 102c18c The "Remove" button for ranks has been fixed.
- [x] df261c6 Issues on Me with the parser have been fixed.
- [x] 4c5e75c Post actions will now appear correctly again.
- [x] 7443957 A bug with the index pagination has been fixed.
- [x] a8ce9f3 Fix a bug with removing users.
- [x] 3ace1ea The notification link is now displayed in the right font in responsive mode.
- [x] 122f9df All available post actions are now displayed correctly again.
- [x] e50226b The backstage_message() function won't give errors anymore.
- [x] c274003 Exclude second part when topic is moved.
- [x] 6cbeecf The styling of the search bar in the navbar has been fixed.
- [x] 42a4535 Fixes editor buttons not working properly.
- [x] 4ef80b91 Notification now can be managed again.
- [x] d46ff2b9 Guest can now see content again.
- [x] aa24eafd Fix for the revamped prune page.

# Thanks to
These are the awesome people that made this possible;

@cyboulette, @invisiplous, @ListenerJubatus, @oldhobbyist, @quy, @rolfpancake, @Sortova, @studio384

And the GetLuna community.
